content/chapitres/projet-1.0.adoc

```diff
@@ -1,48 +1,108 @@
 
-Quel est le livrable ? Définition la production pour le savoir
+== Production pour du HTML ?
 
-== Projet 1.0 : Production
+* *But* : héberger un site web de type statique (HTML+CSS etc.)
 
-Comment héberger un site web de type statique (HTML+CSS etc.)
+* Il faut un service qui permet :
+** (MUST) De servir des fichier statiques via HTTP
+** (NICE) Active le HTTPS avec des certificats et des noms de domaines
 
 == Say Hello to Netlify
 
-Intro à Netlify
+image::Netlify_logo.svg[width=400]
 
-== Exercice Netlify
+https://www.netlify.com/[]
 
-* Depuis le dépôt de l'exercice précédent
-* On ajoute un hugo de base
-* Créer un projet Netlify + config hugo
-* Pousser sur master => Hop magie
+== Netlify : Kézako ?
+
+* Platforme hébergée dans le cloud
+* Offres de services gratuites et payantes en fonction des besoins
+** L'offre gratuite est suffisante pour notre besoin
+* Concept de `git push` -> `netlify build` -> `netlify deploy`
+** Déjà vu ?
 
 == Github Actions vs. Netlify
 
-* Netlify est une forme de CI spécialisé
-* Github action peut aussi pousser dans Netlify (CLI, etc.)
-* Problème : lequel choisir ?
-** La question elle est vite répondue jeune étudiant-entrepreneur : que faire si le test échoue ?
-** Donc Github Actions
+* Netlify est une forme de CI/CD spécialisé
+* Github Actions peut aussi pousser dans Netlify (CLI, etc.)
+* Lequel choisir ?
+** La question elle est vite répondue jeune étudiant-entrepreneur : Github Actions
+** Pourquoi ? Pour rester homogènes et bénéficier des outils de GitHub
+
+== Préparation : CLI Netlify
+
+* Authentifiez-vous sur https://www.netlify.com/[Netlify] (bouton "Log In" puis "GitHub")
+** ⚠️ La 1ère fois vous devrez autoriser Nelify à accéder à votre compte GitHub
+
+* Dans link:https://gitpod.io#https://github.com/cicd-lectures/demoapp[l'environnement GitPod,window=_blank],
+retournez dans `/workspace/exercice-makefile`
+
+* Initialisez l'environnement pour utiliser la CLI Netlify :
+** `netlify login` (https://docs.netlify.com/cli/get-started/#obtain-a-token-via-the-command-line[Documentation,window=_blank])
+*** Si besoin, choissez "Ouvrir dans une nouvelle fenêtre"
+** `netlify status` (https://cli.netlify.com/commands/status[Documentation,window=_blank])
+** `netlify sites:list` (https://cli.netlify.com/commands/sites[Documentation,window=_blank])
+
+== Créer un site avec la CLI Netlify
+
+* Créez un nouveau "site" en suivant les étapes suivantes avec la commande `netlify site:<xxx>` :
+** `netlify sites:list` (https://cli.netlify.com/commands/sites[Documentation,window=_blank])
+** `netlify sites:create` (https://cli.netlify.com/commands/sites[Documentation,window=_blank])
+*** Laissez l'option "Site Name" à vide pour obtenir un nom aléatoire
+
+* Configurez les variables d'environnements pour Netlify:
+** `export NETLIFY_AUTH_TOKEN=XXX` (récupérez `XXX` dans le fichier `~/.netlify/config.json`)
+** `export NETLIFY_SITE_ID=YYY` (récupérez `YYY` à l'aide de la commande `netlify sites:list`)
```

Comment:
Why do we ask them to setup netkify env variables in the gidpod box if they already run `netlify login` ? 🤔

Reply:
-> site_id is to avoid the interactive questions when doing `netlify deploy` on the next slide.
-> auth toekn is to regroup the "fetch an export" variables now, so during the project thet might search on a single slide.